app/forms/idv/ssn_format_form.rb

```diff
@@ -18,12 +16,12 @@ def initialize(incoming_ssn)
       @updating_ssn = ssn.present?
     end
 
-    def submit(params)
-      consume_params(params)
+    def submit(ssn:)
+      @ssn = ssn
 
       FormResponse.new(
         success: valid?,
-        errors: errors,
+        errors:,
         extra: {
           pii_like_keypaths: [
             [:same_address_as_id],
```

Comment:
is `:same_address_as_id` necessary here, or is this an artifact of the code doing something else at some point?